api/python/cellxgene_census/pyproject.toml

```diff
@@ -41,12 +41,12 @@ dependencies= [
 
 [project.optional-dependencies]
 experimental = [
-    "torch==2.0.1",
-    "torchdata==0.6.1",
-    "scikit-learn==1.3.1",
-    "scikit-misc==0.2.0",
-    "psutil==5.9.5",
-    "datasets~=2.14",
+    "torch~=2.0",
+    "torchdata~=0.7",
+    "scikit-learn~=1.0",
+    "scikit-misc~=0.3",
+    "psutil~=5.0",
+    "datasets~=2.16",
```

Comment:
Mike suggested `datasets~=2.0` - I assume you need a feature introduced in 2.16?